sdk-actors/src/main/java/io/dapr/actors/runtime/ActorManager.java

```diff
@@ -14,12 +15,13 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
 import java.util.function.Function;
 
 /**
  * Manages actors of a specific type.
  */
-class ActorManager<T extends AbstractActor> {
+public class ActorManager<T extends AbstractActor> {
```

Comment:
Why does this need to be made public? Making this public is a big deal since it increases the API surface.

Reply:
It doesn't, technically. At this point I was just working towards a PoC but locally I now have the underlying ID (reason why I exposed it) provided by the already exposed ActorRuntime.